### ISSUE TITLE
Pinganyun ECS - Node template list location and size display as N/A

### ISSUE THREAD

**Issue by rootwuj** [rancher/rancher#21277]
**Environment information**
Version: rancher/rancher:v2.2-head

**Steps to reproduce :**
Add node template for Pinganyun ECS.

**Result:**
Node template list location and size display as `N/A`.
<img width="1295" alt="image" src="https://user-images.githubusercontent.com/42061003/60488909-b333e300-9cd5-11e9-8276-7fe38a70d5fa.png">

**Comment by davidnuzik** [comment]
@loganhz I do not have Pinganyun credentials. May I be supplied them or could we ask someone else to test this?

Not related, but I did try to enable the driver and I saw errors in rancher logs around this. Also driver name is not friendly in the UI.

![image](https://user-images.githubusercontent.com/45179589/60543238-a6f65880-9cca-11e9-85ca-77ae2cb098f1.png)
![image](https://user-images.githubusercontent.com/45179589/60543255-b07fc080-9cca-11e9-8077-aaca5ee7551d.png)
